chainlit_app/services/feedback_service.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import requests
+
+API_URL = os.getenv("API_URL", "http://localhost:8000")
```

Comment:
We should import this from `config.py`. Say we decided that the default should be `backend` (which is what it is called when using docker-compose) then we'd have to change it all the files.